vtexdocs/devportal
# Fix/header responsiveness

#### What is the purpose of this pull request?

Fix the header and add the hamburger menu icon.

#### What problem is this solving?

The search bar was appearing at incorrect breakpoints. In addition, it was also necessary to add the hamburger menu icon for the breakpoints where the search bar would not appear.

#### How should this be manually tested?

Please compare the [old version](https://elated-hoover-5c29bf.netlify.app/) with the [new version](https://deploy-preview-22--xenodochial-shockley-a99d82.netlify.app/). You can look at the [figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=129%3A28) and see if everything is working properly.

- At sizes smaller than 1024, the search bar should not appear and the hamburger menu icon should appear.
- In sizes larger or equal to 1024, the search bar should appear and the hamburger menu icon should not appear.

#### Screenshots or example usage

#### Types of changes
